app/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -286,6 +288,7 @@ open class HomeAssistantApplication : Application() {
 
         ContextCompat.registerReceiver(this, buttonWidget, screenIntentFilter, ContextCompat.RECEIVER_NOT_EXPORTED)
         ContextCompat.registerReceiver(this, entityWidget, screenIntentFilter, ContextCompat.RECEIVER_NOT_EXPORTED)
+        ContextCompat.registerReceiver(this, cameraWidget, screenIntentFilter, ContextCompat.RECEIVER_NOT_EXPORTED)
```

Comment:
Why add this here? the widget should update once per hour. Additionally I am not seeing it update when the screen turns on but if I tap on the widget I see 2 updates after the screen turns on.

Reply:
If the camera is not registered stays like that, I feel getting the latest camera snapshot.
https://github.com/user-attachments/assets/f9964768-a61a-46d6-9d86-7c06c14cc57f

Reply:
i dont know what the video is supposed to shw but on the surface it looks like its taking over a minute to update?

Reply:
Sorry no, is using the version without the broadcast and then compiling and relaunching the app

Reply:
ok, as this code was moved to #4648 we should move this convo there :)